template/en/default/email/bugmail.html.tmpl

```diff
@@ -38,7 +38,7 @@
               on [% "$terms.bug $bug.id" FILTER bug_link(bug, { full_url => 1, user => to_user }) FILTER none %]
               from [% INCLUDE global/user.html.tmpl user = to_user, who = comment.author %]</b>
           [% END %]
-        <pre>[% comment.body_full({ wrap => 1 }) FILTER quoteUrls(bug, comment, to_user) %]</pre>
+        <pre>[% comment.body_full({ wrap => 1 }) FILTER renderComment(bug, comment) %]</pre>
```

Comment:
Don't we want markdown to render in HTML emails? The &lt;pre> tag would not allow this.

Reply:
Thanks for catching this! I fixed it in the `extensions/BMO/template/en/default/email/bugmail.html.tmpl` but forgot to fix it here.